include/glow/Support/Support.h

```diff
@@ -205,6 +205,20 @@ constexpr char sepChar = ':';
 /// Convert a string to int. \returns the int or Error if problem parsing.
 Expected<int> getIntFromStr(llvm::StringRef input);
 
+/// Reorder vector \p v according to the indices in \p order. Value at index
+/// idx in \p v will end up at the position indicated by the value in \p order
+/// at index idx.
+template <class T>
+void vectorReorder(std::vector<T> &v, std::vector<size_t> const &order) {
+  for (int s = 1, d; s < order.size(); ++s) {
+    for (d = order[s]; d < s; d = order[d])
+      ;
+    if (d == s)
+      while (d = order[d], d != s)
+        std::swap(v[s], v[d]);
```

Comment:
nit: Normal coding style is to always use curly brackets even when there's 1 line in the body.